Makefile

```diff
@@ -0,0 +1,27 @@
+VERSION ?= unknown
+
+# Packages
+TOP_PACKAGE_DIR := github.com/diskordanz
+PACKAGE_LIST := weather_notifier
+
+all: deps test build
+
+test:
+	go test -v --cover --race -short `glide novendor | grep -v ./proto`
+
+build:
+	CGO_ENABLED=0 go build -a -ldflags '-s -X main.serviceVersion=$(VERSION)' -installsuffix cgo -o main ./cmd/notifier/
+
+deps:
+	glide install
+
+lint: 
+	@for p in $(PACKAGE_LIST); do \
```

Comment:
i think you can use smth like `golint $$(go list ./... | grep -v /vendor/)` instead of your `for..` loop